stylesheets/commons/Colours.less

```diff
@@ -676,6 +676,15 @@ ul.nav-tabs li.game-wiiu,
 	color: var( --clr-california-80, #f9dec7 );
 }
 
+/* Deadlock */
+.deadlock-amberhand-text {
+	color: #c09b54;
+}
+
+.deadlock-sapphireflame-text {
+	color: #5c7ae6;
```

Comment:
same here

Reply:
![image](https://github.com/user-attachments/assets/2751ce26-8224-47e8-b959-567b5af51a80)
![image](https://github.com/user-attachments/assets/6608d595-8e75-47ab-bd19-0b32feaac9ac)

From a quick preview through inspecting the page, colours seem fine to me on both modes